src/pages/index.js

```diff
@@ -163,27 +180,52 @@ profileEditButton.addEventListener("click", () => {
 });
 
 cardModalbtn.addEventListener("click", () => {
-  editModalNameInput.value = profileName.textContent;
-  editModalDescriptionInput.value = profileDescription.textContent;
   openModal(cardModal);
 });
 
+avatarEditButton.addEventListener("click", () => {
+  openModal(avatarModal);
+});
+
+avatarForm.addEventListener("submit", handleAvatarSubmit);
 profileForm.addEventListener("submit", handleEditForSubmit);
 cardForm.addEventListener("submit", handleAddCardSubmit);
-
-initialCards.forEach((card) => {
-  const cardElement = getCardElement(card);
-  cardsList.prepend(cardElement);
-});
+deleteForm.addEventListener("submit", handleDeleteSubmit);
+cancelButton.addEventListener("click", () => closeModal(deleteModal));
 
 enableValidation(settings);
 
+function handleEditForSubmit(e) {
+  handleSubmit(
+    () =>
+      api.editUserInfo({
+        name: editModalNameInput.value,
+        about: editModalDescriptionInput.value,
+      }),
+    e
```

Comment:
` e`


You've forgotten to add the updating in `then` inside the request funcitions

```js

function handleEditForSubmit(e) {
  handleSubmit(
    () =>
      api
        .editUserInfo({
          name: editModalNameInput.value,
          about: editModalDescriptionInput.value,
        })
        .then(() => {
          // here you update the page
          closeModal(editModal);
        }),
    e
  );
}

```